x/perp/v2/keeper/liquidate.go

```diff
@@ -222,12 +240,12 @@ func (k Keeper) liquidate(
 		liquidationResponse.FeeToLiquidator,
 	)
 
-	ecosystemFundFee = sdk.NewCoin(
+	insuranceFundFee = sdk.NewCoin(
```

Comment:
Can we keep consistency in the names? It would be confusing if we use `ecosystemFund` in some places in the code and `insuranceFund` in other places.

Reply:
- Moving this to separate PR  and adding the comment to ticket #1468